META.json

```diff
@@ -32,7 +32,7 @@
    },
    "generated_by": "Julien Rouhaud",
    "meta-spec": {
-      "version": "1.0.0",
+      "version": "1.0.1",
```

Comment:
This is confusing, but it's the version of the pgxn meta spec, not the extension.  The version is automatically updated in the releaze-zip Makefile rule.